console/src/group/ontology.tsx

```diff
@@ -249,7 +249,7 @@ export const useCreateFromSelection = (): ((
       if (selection.parent == null) return;
       const resourcesToGroup = getResourcesToGroup(selection);
       const prevNodes = Tree.deepCopy(nodes);
-      let nextNodes = Tree.setNode({
+      Tree.setNode({
```

Comment:
This function produces a shallow copy of the tree, which is why we had the assignment here. Can you add it back?